ra/ra.go

```diff
@@ -480,18 +488,18 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, reque
 	// Get a pending Auth first so we can get our ID back, then update with challenges
 	authz, err = ra.SA.NewPendingAuthorization(ctx, authz)
 	if err != nil {
-		// InternalServerError since the user-data was validated before being
+		// berrors.InternalServer since the user-data was validated before being
```

Comment:
This comment and the one below are inaccurate, since the actual name of the error is berrors.InternalServerError.